### PR TITLE
Centralize the rootContainer and Worked on the FIlter Panel

### PR DESCRIPTION
**Pull request by egbobright**
Increased the width of the id yourDiv from 60% to 90%. This helped clear out the jumbled figures in the chart.
 2. Centralize the rootContainer by applying the margin-left property. This should make it more visible and accessible.
 3. Worked on the Filter Panel by:
 a. changing the background color and the text color. 
b. changing the Results length and 
c. applied border-radius accross all select and input tag.

### CODE REVIEW

**Comment by heralden** [comment]
This is a great start, but I'm not a fan of the new background colour and button hover colour (there's too much contrast compared to the rest of the elements).

I was also hoping to see a different layout of the filter panel. I think it can be made more compact, perhaps like this (I moved things around in gimp). Do you think you could work on this?

compact-filter-panel

**Comment by egbobright** [comment]
OK. I'll work on the Filter Panel again.

Thank you for the feedback.

**Comment by egbobright** [comment]
Kindly check the improvement on the pull request I made earlier.
I have reverted to the original background color but I worked on the display property in order to bring it close to gimp prototype.

**Comment by egbobright** [comment]
This is how it looks now:
new_go_term

File: src/style.less [hunk 1]
```diff
@@ -13,6 +13,7 @@
 	.rootContainer {
 		border: 1px solid black;
 		padding: 20px;
+		margin-left: 105px;
```
Comment (heralden) [inline src/style.less]: This visualization is for use in Bluegenes, so the extra margin would add a lot of empty space when the viz is displayed inside Bluegenes. I agree that it makes it nicer to look at when running `npm run dev`, but a nicer displayer for each of the tools is something we want to work on in the future.

Sorry that this hasn't been specified properly!

To summarise, please remove the margin-left property.

**Review by heralden — CHANGES_REQUESTED** [review]
It looks fine now! There's just one more thing before I merge this; please see my comment.

**Comment by egbobright** [comment]
Many thanks for your review. The change has been implemented on the rootContainer class.

**Review by heralden — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**